dist/package/openSUSE-release-tools.spec

```diff
@@ -16,6 +16,17 @@
 #
 
 
+%if 0%{?suse_version} > 1500 || 0%{?sle_version} >= 150100
```

Comment:
I would prefer to get rid of python2 (and only test python3) - and require osc 0.165. But as an intermediate step, I'm fine with this.

Reply:
Sure, although keep in mind we package staging plugins and such for Leap 15.0 which has osc python 2.

Reply:
Not in openSUSE:Tools:
https://build.opensuse.org/package/binary/openSUSE:Tools/osc/SLE_15/x86_64/osc-0.165.0-250.1.noarch.rpm

Reply:
Then the condition should be updated I suppose.

Reply:
The problem is as soon as switch the interpreter of tools to /usr/bin/python3, the osclib requires both python version. And as I said: I rather get rid of python2 and say sorry for everyone with sle12/leap 42

Reply:
Not disagreeing with long-term plan, but the same applies to osc which is doing the split. Simply drop once all deployments are working on python3.